modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -112,8 +122,48 @@ command and can be used to further customize the mirroring configuration.
 . If you used the `--manifests-only` flag in the previous step and want to mirror
 only a subset of the content:
 
-.. Modify the list of images in your `mapping.txt` file to your
-specifications.
+.. Modify the list of images in your `mapping.txt` file to your specifications. If
+you are unsure of the exact names and versions of the subset of images you want
+to mirror:
```

Comment:
Probably fine as is, but I had to read this twice, because I was expecting an end to the sentence, or a second option (that you are sure).

Reply:
Yeah, at the last sentence I moved this part up from where I previously had it, at the beginning of the "Run the sqlite3 tool..." substep... Still working the best way to show this.

Reply:
Changed to the following to be more obvious: "If
you are unsure of the exact names and versions of the subset of images you want
to mirror, use the following steps to find them:"